pages/common/pip-inspect.md

```diff
@@ -0,0 +1,29 @@
+# pip inspect
+
+> Inspect the Python environment and produce a report in JSON format.
+> Shows detailed information about installed packages and their metadata.
```

Comment:
```suggestion
```
These second lines don't really provide anything useful that couldn't be explained through an example.